mlir/lib/Conversion/MemRefToLLVM/MemRefToLLVM.cpp

```diff
@@ -520,9 +520,7 @@ struct GlobalMemrefOpLowering
         global, arrayTy, global.getConstant(), linkage, global.getSymName(),
         initialValue, alignment, *addressSpace);
     if (!global.isExternal() && global.isUninitialized()) {
-      Block *blk = new Block();
```

Comment:
Oh these raw uses of `new` always irked me deeply! Glad to see more that are gone